utilities/utils.js

```diff
@@ -5,6 +5,6 @@ exports.checkExists = async (table, column, value) => {
     const queryString = format('SELECT * FROM %I WHERE %I = $1;', table, column)
     const output = await db.query(queryString, [value])
     if(output.rows.length === 0){
-        return Promise.reject({status: 404, msg: "Requested ID not found"})
+        return Promise.reject({status: 404, msg: `Requested ${column} not found`})
```

Comment:
point to ponder: to what extent does this allow scope to potentially compromise your db?